### PR TITLE
Release 1.11.0

### PR DESCRIPTION
**Pull request by shanmydell**

### CODE REVIEW

**Review by harshaatdell — APPROVED** [review]

File: content/docs/csidriver/release/powermax.md [hunk 1]
```diff
@@ -15,19 +15,25 @@ description: Release notes for PowerMax CSI driver
...
+
 ### New Features/Changes
 
-- [#1284 - [FEATURE]: Support for Openshift 4.15](https://github.com/dell/csm/issues/1284)
-- [#1285 - [FEATURE]: Remove checks in code for non-supported installs of CSM](https://github.com/dell/csm/issues/1285)
-- [#926 - [FEATURE]: Fixing the linting, formatting and vetting issues](https://github.com/dell/csm/issues/926)
+- [#1308 - [FEATURE]: NVMe TCP support for PowerMax ](https://github.com/dell/csm/issues/1308)
+- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
+- [#1400 - [FEATURE]: Support for Kubernetes 1.30](https://github.com/dell/csm/issues/1400)
+- [#1082 - [FEATURE]:CSM Resiliency support for PowerMax ](https://github.com/dell/csm/issues/1082)
```
Comment (gallacher) [inline content/docs/csidriver/release/powermax.md:24]: Missing space

File: content/docs/deployment/csminstallationwizard/src/templates/operator/csm-powerstore-1.11.0.template [hunk 2]
```diff
@@ -0,0 +1,195 @@
+#
+#
+# Copyright © 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (gallacher) [inline content/docs/deployment/csminstallationwizard/src/templates/operator/csm-powerstore-1.11.0.template:3]: Should be 2024

File: content/docs/prerequisites/_index.md [hunk 3]
```diff
@@ -54,40 +54,42 @@ Container Storage Modules (CSM) does not officially support specific operating s
...
+| CSI PowerFlex      | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
+| CSI PowerFlex      | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ✔ 1.8.0       |
+| CSI PowerStore     | 2.11.0  | ❌                | ❌             | ❌                | ✔ 1.10.0      |
+| CSI PowerStore     | 2.10.0  | ❌                | ❌             | ❌                | ✔ 1.9.0       |
+| CSI PowerStore     | 2.9.0   | ❌                | ❌             | ❌                | ✔ 1.8.0       |
+| CSI PowerMax       | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerMax       | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1        | ✔ 1.8.1           | ❌            |
+| CSI PowerMax       | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ❌            |
+| CSI Unity XT       | 2.11.0  | ❌                | ❌             | ❌                | ❌            |
+| CSI Unity XT       | 2.10.0  | ❌                | ❌             | ❌                | ❌            |
```
Comment (lukeatdell) [inline content/docs/prerequisites/_index.md:93]: looks like this should be 2.10.1

File: content/docs/prerequisites/_index.md [hunk 4]
```diff
@@ -54,40 +54,42 @@ Container Storage Modules (CSM) does not officially support specific operating s
...
-| CSI Unity XT       | 2.9.0   | ❌                | ❌             | ❌                | ❌             |
-| CSI Unity XT       | 2.8.0   | ❌                | ❌             | ❌                | ❌             |
+| CSI PowerScale     | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerScale     | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
+| CSI PowerScale     | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ✔ 1.8.0       |
+| CSI PowerFlex      | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerFlex      | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
+| CSI PowerFlex      | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ✔ 1.8.0       |
+| CSI PowerStore     | 2.11.0  | ❌                | ❌             | ❌                | ✔ 1.10.0      |
+| CSI PowerStore     | 2.10.0  | ❌                | ❌             | ❌                | ✔ 1.9.0       |
```
Comment (lukeatdell) [inline content/docs/prerequisites/_index.md:87]: looks like this should be 2.10.1

File: content/docs/prerequisites/_index.md [hunk 5]
```diff
@@ -54,40 +54,42 @@ Container Storage Modules (CSM) does not officially support specific operating s
...
-| CSI PowerMax       | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ❌             |
-| CSI PowerMax       | 2.8.0   | ✔ 1.8.0           | ✔ 1.6.0        | ✔ 1.6.0           | ❌             |
-| CSI Unity XT       | 2.10.1  | ❌                | ❌             | ❌                | ❌             |
-| CSI Unity XT       | 2.9.0   | ❌                | ❌             | ❌                | ❌             |
-| CSI Unity XT       | 2.8.0   | ❌                | ❌             | ❌                | ❌             |
+| CSI PowerScale     | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerScale     | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
+| CSI PowerScale     | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ✔ 1.8.0       |
+| CSI PowerFlex      | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerFlex      | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
```
Comment (lukeatdell) [inline content/docs/prerequisites/_index.md:84]: looks like this should be 2.10.1

File: content/docs/prerequisites/_index.md [hunk 6]
```diff
@@ -54,40 +54,42 @@ Container Storage Modules (CSM) does not officially support specific operating s
...
-| CSI PowerStore     | 2.9.0   | ❌                | ❌             | ❌                | ✔ 1.8.0        |
-| CSI PowerStore     | 2.8.0   | ❌                | ❌             | ❌                | ✔ 1.7.0        |
-| CSI PowerMax       | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1        | ✔ 1.8.1           | ❌             |
-| CSI PowerMax       | 2.9.0   | ✔ 1.9.0           | ✔ 1.7.0        | ✔ 1.7.0           | ❌             |
-| CSI PowerMax       | 2.8.0   | ✔ 1.8.0           | ✔ 1.6.0        | ✔ 1.6.0           | ❌             |
-| CSI Unity XT       | 2.10.1  | ❌                | ❌             | ❌                | ❌             |
-| CSI Unity XT       | 2.9.0   | ❌                | ❌             | ❌                | ❌             |
-| CSI Unity XT       | 2.8.0   | ❌                | ❌             | ❌                | ❌             |
+| CSI PowerScale     | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
+| CSI PowerScale     | 2.10.0  | ✔ 1.10.0          | ✔ 1.8.0        | ✔ 1.8.0           | ✔ 1.9.0       |
```
Comment (lukeatdell) [inline content/docs/prerequisites/_index.md:81]: looks like this should be 2.10.1

File: content/docs/deployment/rpm/modules/installation/authorization/authorization.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ description: >
 ---
 
 {{% pageinfo color="primary" %}}
-The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in a future release. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
+The CSM Authorization RPM will be deprecated in a future release. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
```
Comment (lukeatdell) [inline content/docs/deployment/rpm/modules/installation/authorization/authorization.md:9]: possibly revert this change since the old description appears to be more accurate.

File: content/docs/deployment/offline/drivers/_index.md [hunk 8]
```diff
@@ -83,30 +83,30 @@ bash scripts/csm-offline-bundle.sh -c
...
    dellemc/csipowermax-reverseproxy:v2.6.0
-   dellemc/csi-powermax:v2.10.1
-   dellemc/csi-powerstore:v2.10.1
-   dellemc/csi-unity:v2.10.1
-   dellemc/csi-vxflexos:v2.10.1
-   dellemc/csm-authorization-sidecar:v1.10.0
+   dellemc/csi-powermax:v2.10.0
+   dellemc/csi-powerstore:v2.10.0
+   dellemc/csi-unity:v2.10.0
+   dellemc/csi-vxflexos:v2.10.0
```
Comment (lukeatdell) [inline content/docs/deployment/offline/drivers/_index.md:92]: update all these driver images to v2.11.0?

**Review by lukeatdell — COMMENTED** [review]
this isn't a complete review.
It covers from `content/docs/support/troubleshooting/_index.md` to `content/docs/deployment/helm/modules/installation/replication/install-repctl.md` in the tree on the github review UI for file changes.

**Review by hoppea2 — APPROVED** [review]

**Review by gallacher — APPROVED** [review]

**Review by santhoshatdell — APPROVED** [review]

**Review by lukeatdell — APPROVED** [review]

**Review by anandrajak1 — APPROVED** [review]
